### PR TITLE
Add Polish Language Support

### PR DESCRIPTION
**Pull request by memerson12**
Polish language using translations [provided by @MadoBrick in the discord ](https://discord.com/channels/1262887973154848828/1272273974889550024/1272435221513506899)

### CODE REVIEW

**Review by tildejustin — CHANGES_REQUESTED** [review]
language file filenames should be all lowercase as they match to resource locations in game

**Comment by memerson12** [comment]
> language file filenames should be all lowercase as they match to resource locations in game

Updated. Didn't realize that Minecraft didn't use the standard. I just pulled the locale code from here https://docs.oracle.com/cd/E23824_01/html/E26033/glset.html

**Comment by KingContaria** [comment]
Thanks for the PR! :)
You'll need one more person verifying your translations, please check the newly added translations workflow: https://github.com/KingContaria/seedqueue/issues/66

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
+  "seedqueue.menu.clearing": "Czyszczę SeedQueue...",
+
+  "seedqueue.menu.crash.title": "SeedQueue się zcrashowało!",
+  "seedqueue.menu.crash.description": "Crash przez %s, sprawdź log po więcej informacji.",
+
+  "seedqueue.menu.benchmark.title": "SeedQueue Benchmark",
+  "seedqueue.menu.benchmark.result": "%s resetów w %ss (%s RPS)",
+
+  "seedqueue.menu.config.useWall.notAvailable": "§c§oNie Dostępne",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:10]:
```suggestion
  "seedqueue.menu.config.useWall.notAvailable": "§c§oNiedostępne",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
+{
+  "seedqueue.menu.clearing": "Czyszczę SeedQueue...",
+
+  "seedqueue.menu.crash.title": "SeedQueue się zcrashowało!",
+  "seedqueue.menu.crash.description": "Crash przez %s, sprawdź log po więcej informacji.",
+
+  "seedqueue.menu.benchmark.title": "SeedQueue Benchmark",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:7]:
```suggestion
  "seedqueue.menu.benchmark.title": "Test Sprawności SeedQueue",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  "seedqueue.menu.benchmark.result": "%s resetów w %ss (%s RPS)",
+
+  "seedqueue.menu.config.useWall.notAvailable": "§c§oNie Dostępne",
+  "seedqueue.menu.config.useWall.notAvailable.tooltip": "Używanie Ekranu Ściany potrzebuje zainstalowania tych modów::",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:11]:
```suggestion
  "seedqueue.menu.config.useWall.notAvailable.tooltip": "Używanie Ekranu Ściany potrzebuje zainstalowania tych modów:",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  "seedqueue.menu.config.useWall.notAvailable.tooltip.3": " - WorldPreview",
+
+  "seedqueue.menu.config.showAdvancedSettings.confirm.title": "Czy na pewno chcesz włączyć Ustawienia Zaawansowane?",
+  "seedqueue.menu.config.showAdvancedSettings.confirm.message": "Używaj tych opcji tylko jeśli jesteś pewien co robią albo gdy jesteś informowany przez kogoś kto wie co robi!",
+
+  "seedqueue.menu.keys": "Skróty Klawiszowe Ściany",
+  "seedqueue.menu.keys.configure": "Konfiguruj...",
+  "seedqueue.menu.keys.primary": "Klawisz Główny",
+  "seedqueue.menu.keys.secondary": "§7§oKlawisze Pomocnicze",
+  "seedqueue.menu.keys.secondary.tooltip": "Kiedy zostanie naciśnięty klawisz główny, gra sprawdzi, czy wszystkie klawisze pomocnicze są również naciśnięte, zanim aktywuje się skrót klawiaturowy.",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:23]:
```suggestion
  "seedqueue.menu.keys.secondary.tooltip": "Kiedy klawisz główny zostanie naciśnięty, gra sprawdzi, czy wszystkie klawisze pomocnicze są również naciśnięte, zanim aktywuje się skrót klawiaturowy.",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  "seedqueue.menu.keys.blocking": "§7§oKlawisze Blokujące",
+  "seedqueue.menu.keys.blocking.tooltip": "Kiedy zostanie naciśnięty klawisz główny, gra sprawdzi, czy żadne klawisze blokujące §lnie§r są naciśnięte, zanim aktywuje się skrót klawiaturowy.",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:25]:
```suggestion
  "seedqueue.menu.keys.blocking.tooltip": "Kiedy klawisz główny zostanie naciśnięty, gra sprawdzi, czy żadne klawisze blokujące §lnie§r są naciśnięte, zanim aktywuje się skrót klawiaturowy.",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+  "seedqueue.key.play": "Zagraj w ten Świat",
+  "seedqueue.key.lock": "Użyj Kłódki na Świat",
+  "seedqueue.key.reset": "Zresetuj Świat",
+  "seedqueue.key.resetAll": "Zresetuj wszystkie Światy",
+  "seedqueue.key.focusReset": "Skupiony Reset",
+  "seedqueue.key.resetRow": "Zresetuj Rząd",
+  "seedqueue.key.resetColumn": "Zresetuj Kolumnę",
+  "seedqueue.key.playNextLock": "Zagraj w Następną Kłódkę",
+  "seedqueue.key.startBenchmark": "Zacznij Benchmark",
+  "seedqueue.key.cancelBenchmark": "Skończ Benchmark",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:40]:
```suggestion
  "seedqueue.key.startBenchmark": "Zacznij Test Sprawności",
  "seedqueue.key.cancelBenchmark": "Zakończ Test Sprawności",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+  "speedrunapi.config.seedqueue.category.queue": "Ustawnienia Kolejki",
+  "speedrunapi.config.seedqueue.category.chunkmap": "Ustawienia Chunkmap",
+  "speedrunapi.config.seedqueue.category.wall": "Ustawienia Ekranu Ściany",
+  "speedrunapi.config.seedqueue.category.performance": "Ustawnienia Wydajności",
+  "speedrunapi.config.seedqueue.category.advanced": "Ustawienia Zaawansowane",
+  "speedrunapi.config.seedqueue.category.threading": "Ustawienia Wątków",
+  "speedrunapi.config.seedqueue.category.experimental": "Ustawienia Eksperymentalne",
+  "speedrunapi.config.seedqueue.category.debug": "Ustawienia Debugujące",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:49]:
```suggestion
  "speedrunapi.config.seedqueue.category.debug": "Ustawienia Odpluskwiania",
```
😄

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+  "speedrunapi.config.seedqueue.option.maxCapacity": "Maksymalna Ilość Seedów w Kolejce",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:51]: Minecraft tłumaczy na `ziarno`, ale jak nie tłumaczymy `SeedQueue` to można zostać przy tym

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+  "seedqueue.menu.keys.secondary.tooltip": "Kiedy zostanie naciśnięty klawisz główny, gra sprawdzi, czy wszystkie klawisze pomocnicze są również naciśnięte, zanim aktywuje się skrót klawiaturowy.",
+  "seedqueue.menu.keys.blocking": "§7§oKlawisze Blokujące",
+  "seedqueue.menu.keys.blocking.tooltip": "Kiedy zostanie naciśnięty klawisz główny, gra sprawdzi, czy żadne klawisze blokujące §lnie§r są naciśnięte, zanim aktywuje się skrót klawiaturowy.",
+  "seedqueue.menu.keys.secondary_list": "§7§oPomocniczy:",
+  "seedqueue.menu.keys.blocking_list": "§7§oBlokujący:",
+
+  "seedqueue.key.categories.builtin": "Wbudowane Skróty Klawiszowe",
+  "seedqueue.key.categories.custom": "Niestandardowe Skróty Klawiszowe",
+  "seedqueue.key.play": "Zagraj w ten Świat",
+  "seedqueue.key.lock": "Użyj Kłódki na Świat",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:32]:
```suggestion
  "seedqueue.key.lock": "Przypnij Świat",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+  "seedqueue.key.reset": "Zresetuj Świat",
+  "seedqueue.key.resetAll": "Zresetuj wszystkie Światy",
+  "seedqueue.key.focusReset": "Skupiony Reset",
+  "seedqueue.key.resetRow": "Zresetuj Rząd",
+  "seedqueue.key.resetColumn": "Zresetuj Kolumnę",
+  "seedqueue.key.playNextLock": "Zagraj w Następną Kłódkę",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:38]:
```suggestion
  "seedqueue.key.playNextLock": "Zagraj w Następny Przypięty Świat",
```

File: src/main/resources/assets/seedqueue/lang/pl_pl.json [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+  "speedrunapi.config.seedqueue.option.previewFPS": "FPS Podglądu Świata",
+  "speedrunapi.config.seedqueue.option.previewFPS.description": "Limit liczby klatek na sekundę podglądu na ekranie ściany.\nJest on względny do rzeczywistej prędkości renderowania na ekranie ściany. Z tego powodu, ta opcja może działać nieoczekiwanie, gdy liczba klatek na sekundę na ekranie ściany jest ustawiona na \"Bez Limitu",
+  "speedrunapi.config.seedqueue.option.previewFPS.value": "%s fps",
+  "speedrunapi.config.seedqueue.option.previewFPS.value.255": "Bez Limitu",
+  "speedrunapi.config.seedqueue.option.backgroundPreviews": "Podglądy Świata w Tle",
+  "speedrunapi.config.seedqueue.option.backgroundPreviews.description": "Tworzy podglądy w tle przed rozpoczęciem ich renderowania, co umożliwia płynniejsze przejście podczas ładowania nowego podglądu na ekran ściany.",
+  "speedrunapi.config.seedqueue.option.freezeLockedPreviews": "Zamróź Podglądy z Kłódką",
+  "speedrunapi.config.seedqueue.option.freezeLockedPreviews.description": "Jeśli ta opcja jest włączona, zablokowane podglądy nie będą otrzymywać nowych chunków, a ich podgląd nie będzie aktualizowany",
```
Comment (MiniaczQ) [inline src/main/resources/assets/seedqueue/lang/pl_pl.json:91]:
```suggestion
  "speedrunapi.config.seedqueue.option.freezeLockedPreviews": "Zamróź Przypięte Podglądy",
  "speedrunapi.config.seedqueue.option.freezeLockedPreviews.description": "Jeśli ta opcja jest włączona, przypięte podglądy nie będą otrzymywać nowych chunków i ich podgląd nie będzie aktualizowany",
```

**Review by MiniaczQ — COMMENTED** [review]
More or less correct, it's missing some entries compared to https://github.com/KingContaria/seedqueue/blob/main/src/main/resources/assets/seedqueue/lang/en_us.json

**Comment by memerson12** [comment]
> More or less correct, it's missing some entries compared to https://github.com/KingContaria/seedqueue/blob/main/src/main/resources/assets/seedqueue/lang/en_us.json

That's because the PR was made very early on and some translation keys were added later. I went ahead and committed your suggested changes.

**Comment by KingContaria** [comment]
I will take this as a verification and probably merge the translations with 1.4

**Review by KingContaria — APPROVED** [review]

**Pull request merged**